docs/userguide/resource-propagating.md

```diff
@@ -0,0 +1,155 @@
+# Resource Propagating
+
+The [PropagationPolicy](https://github.com/karmada-io/karmada/blob/master/pkg/apis/policy/v1alpha1/propagation_types.go#L13) and [ClusterPropagationPolicy](https://github.com/karmada-io/karmada/blob/master/pkg/apis/policy/v1alpha1/propagation_types.go#L292) APIs are provided to propagate resources. For the differences between the two APIs, please see [here](../frequently-asked-questions.md#what-is-the-difference-between-propagationpolicy-and-clusterpropagationpolicy).
+
+Here, we use PropagationPolicy as an example to describe how to propagate resources.
+
+## Before you start
+
+[Install Karmada](../installation/installation.md) and prepare the [karmadactl command-line](../installation/install-kubectl-karmada.md) tool.
+
+## Deploy a simplest multi-cluster Deployment
+
+### Create a PropagationPolicy object
+
+You can propagate a Deployment by creating a PropagationPolicy object defined in a YAML file. For example, this YAML
+ file describes a Deployment object named nginx under default namespace need to be propagated to member1 cluster:
+
+```yaml
+# propagationpolicy.yaml
+apiVersion: policy.karmada.io/v1alpha1
+kind: PropagationPolicy
+metadata:
+  name: example-policy # The default namespace is `default`.
+spec:
+  resourceSelectors:
+    - apiVersion: apps/v1
+      kind: Deployment
+      name: nginx # If no namespace is specified, the namespace is inherited from the parent object scope.
+  placement:
+    clusterAffinity:
+      clusterNames:
+        - member1
+```
+
+1. Create a propagationPolicy base on the YAML file:
+```shell
+kubectl apply -f propagationpolicy.yaml
+```
+2. Create a Deployment nginx resource:
+```shell
+kubectl create deployment nginx --image nginx
+```
+> Note: The resource exists only as a template in karmada. After being propagated to a member cluster, the behavior of the resource is the same as that of a single kubernetes cluster.
```

Comment:
How to realize "The resource exists only as a template in karmada" ? stop kube-controller-manager in fed member?

Reply:
@XiShanYongYe-Chang @Tingtal

Reply:
We only enable part of controllers with `Kubernetes Controllers`, you can refer to [here](https://github.com/karmada-io/karmada/blob/master/docs/userguide/configure-controllers.md#kubernetes-controllers).

So when we create a resource in `karmada-apiserver`, take Deployment for example, ReplicaSet and Pod will not be generated.

Reply:
But in practice, some infra services in fed cluster are deployed as deployments and pods are needed.

Reply:
`karmada-controller-manager` is deployed in the host cluster, not `karmada-apiserver`.

Reply:
kube-apiserver and kube-etcd in host cluster, karmada-apiserver and karmada-etcd in fed cluster.   Is that so?

Reply:
`karmada-apiserver` and `karmada-etcd` are also deployed in the host cluster with namespace `karmada-system`. We can get them all with the `karmada-host` context with the command: `kubectl --kubeconfig /root/.kube/karmada.config --context  karmada-host get po -A`.

For resources that created in the `karmada-apiserver`, which are only resource templates. We can get them by `karmada-apiserver` context.